client/setec/fileclient.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package setec
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/tailscale/setec/types/api"
+)
+
+// FileClient is an implementation of the StoreClient interface that vends
+// secrets from a static collection of data stored locally on disk.
+//
+// This is intended for use in bootstrapping and deployments without access to
+// a separate secrets server.
+type FileClient struct {
+	path string                      // local filesystem path, for diagnostics
+	db   map[string]*api.SecretValue // :: secret name → static version
+}
+
+// NewFileClient constructs a new FileClient using the contents of the
+// specified local file path. The file must contain a JSON object having the
+// following structure:
+//
+//	{
+//	   "secret-name-1": {
+//	      "secret": {"Value": "b3BlbiBzZXNhbWU=", "Version": 1}
+//	   },
+//	   "secret-name-2": {
+//	      "secret": {"Value": "eHl6enk=", "Version": 5}
+//	   },
+//	   ...
+//	}
+//
+// The secret values are encoded as base64 strings. A cache file written out by
```

Comment:
I have not yet, but could easily add a subcommand to the CLI to generate these files.